text/0199-support-elastic-common-schema-in-opentelemetry.md

```diff
@@ -0,0 +1,289 @@
+# Merge Elastic Common Schema with OpenTelemetry Semantic Conventions
+
+## Introduction
+
+This proposal is to merge the Elastic Common Schema (ECS) with the OpenTelemetry Semantic Conventions (SemConv) and provide full interoperability in OpenTelemetry component implementations. We propose to implement this by aligning the OpenTelemetry Semantic Conventions with [ECS FieldSets](https://www.elastic.co/guide/en/ecs/current/ecs-field-reference.html#ecs-fieldsets) and vice versa where feasible. The long-term goal is to achieve convergence of ECS and OTel Semantic Conventions into a single open schema (e.g. OpenTelemetry Common Schema).
```

Comment:
> OpenTelemetry Common Schema

My understanding based on [this comment](https://github.com/open-telemetry/oteps/pull/222#issuecomment-1424261453) is that "OTEL semconv truly is a "successor" to existing ECS". Let's call this out here. We're not creating a new OpenTelemetry Common Schema, we're merging ECS into Otel Semconv such that Otel Semconv is the successor.

Reply:
@jack-berg Does the recent change resolve your comment?